mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/MapboxDirections.java

```diff
@@ -131,6 +131,9 @@ public void cancelCall() {
    */
   public static class Builder<T extends Builder> extends MapboxBuilder {
 
+    private static final int TWO_COORDINATES = 2;
```

Comment:
@cammace I don't like the approach of having this kind of _all-constant_ classes. Constants should be in their pertinent classes, basically because it's where they belong and provide more context. Actually, [using an _all-constant_ class approach is a well known antipattern](https://softwareengineering.stackexchange.com/questions/49572/is-it-a-bad-practice-to-have-an-interface-to-define-constants#answer-49579).